packages/button/_button.scss

```diff
@@ -55,6 +63,21 @@
         color: $govuk-text-colour;
       }
     }
+
+    // The following adjustments do not work for <input type="button"> as
```

Comment:
So I think `postcss-nested` is doing _crazy things_ here. It's removed the nesting of the `::before` and `::active::before` by shunting them down the file, taking the _immediately preceding_ comment _line_, but leaving the _rest_ of the comment here, divorced from its context.

Not sure what we can do about this in the short term, but… this is not great.

Reply:
As just discussed in person, we should review why we're using `postcss-nested` when @igloosi is back.